codegen/smithy-dafny-codegen-cli/src/main/java/software/amazon/polymorph/CodegenCli.java

```diff
@@ -242,6 +245,14 @@ private static Options getCliOptionsForBuild() {
           .hasArg()
           .build()
       )
+      .addOption(
+        Option
+          .builder()
+          .longOpt("output-go")
+          .desc("<optional> output directory for generated Go files")
```

Comment:
Why is this optional for Go (and Python as well it seems)?

Reply:
Because it is an optional parameter in dafny. If you don't provide this, the code gets generated wherever the dafny file lives.

Reply:
Sure, that's true of all the target languages as well. Just wondering whether there's a reason only Python and Go make it optional.

Reply:
Because Python and Go do it the right way? :) . But yeah the reason was to have a better control - you can provide one if you want but you can also skip it (for prototyping and stuff).